cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -64,7 +65,9 @@ func CreateServiceAccounts(clientset *clientset.Clientset) error {
 
 	for _, sa := range serviceAccounts {
 		if _, err := clientset.CoreV1().ServiceAccounts(metav1.NamespaceSystem).Create(&sa); err != nil {
-			return err
+			if !apierrors.IsAlreadyExists(err) {
+				return err
+			}
```

Comment:
Will calling Update() actually do anything for these service accounts? Based on the creation code above, they just have a name and a namespace, so I thought an Update would just be a no-op. If the service accounts exist, they should already have the same name and namespace, no?